AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js

```diff
@@ -196,26 +199,34 @@ define([
             }
             this.visibility(visibility);
 
-            img = $('[data-image-preview] img');
+            img = $(this.previewImageSelector + ' img');
             if(img.get(0).complete) {
                 this._updateHeight();
             } else {
                 img.load(this._updateHeight.bind(this));
-                this.lastOpenedImage = record._rowIndex;
             }
+            this.lastOpenedImage = this._isInt(record) ? record : record._rowIndex;
         },
 
         /**
          *
          * @private
          */
         _updateHeight: function (){
-            var $preview = $('[data-image-preview]');
-
-            this.height($preview.height() + 'px');// set height
+            this.height($(this.previewImageSelector).height() + 'px');// set height
             this.visibility(this.visibility());// rerender
-            // update scroll if needed
-            $preview.get(0).scrollIntoView({behavior: "smooth", block: "center", inline: "nearest"});
+            this.scrollToPreview();
```

Comment:
Please move this call to `AdobeStockImageAdminUi/view/adminhtml/web/panel.js` next to `this.masonry().setLayoutStyles();`